### PR TITLE
Map skeleton

### PR DESCRIPTION
**Pull request by niklasmaki**

### CODE REVIEW

File: components/quake_map.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import dash_html_components as html
+import dash_core_components as dcc
+import dash_leaflet as dl
+
+from app import app
+
+
+api_base_url = 'https://tile.thunderforest.com/landscape/{z}/{x}/{y}.png?'
+api_url = api_base_url + 'apikey=' + app.server.config['THUNDERFOREST_API_KEY']
```
Comment (llmlks) [inline components/quake_map.py]: Is there a reason these two are defined as separate variables?

File: views/quake_map.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output, State
+
+from app import app
```
Comment (llmlks) [inline views/quake_map.py]: These imports aren't used anywhere

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# Config gile
```
Comment (llmlks) [inline .gitignore]: Minor typo: gile > file

**Review by llmlks — CHANGES_REQUESTED** [review]

**Comment by niklasmaki** [comment]
Made the requested changes

**Comment by niklasmaki** [comment]
I merged the changes already because I need them for another task. If there's still some changes I should make, I'll open another pull request.

**Pull request merged**